cypress/support/config/services.js

```diff
@@ -58,6 +59,7 @@ const services = {
         path: Cypress.env('APP_ENV') === 'live' ? undefined : '/afrique',
         smoke: false,
       },
+      FIX: { path: '/afrique/48465371', smoke: true },
```

Comment:
There only appears to be one working feature index fixture at the moment, how do I generate more?

Reply:
There's another one here `/mundo/noticias-36795069`. Easiest to probably ask Jon to create/find more

Reply:
Aha - so there are a few more already in the repo, but no story promos are rendered because the data in the group does not contain a `strapline`. Should this data contain a strapline, or do we need to be able to render content without a strapline?

Reply:
There were some created in this: https://github.com/bbc/simorgh/pull/3941/files, but have since been moved into the `cpsAssets` directory but I think the IDs should be the same. There may be more though and don't have an answer for you about straplines!

Reply:
The data should contain a strapline however our legacy system has not rendered them before so the editorials teams never really used them.

Reply:
OK great, thanks!